cpu/kinetis/periph/mcg.c

```diff
@@ -176,6 +176,66 @@ static void kinetis_mcg_init_mcgirclk(void)
     }
 }
 
+/**
+ * @brief   Internal helper for performing the actual auto-trim of the reference oscillators
+ *
+ * This is called internally by @ref kinetis_mcg_auto_trim_irc
+ */
+static int kinetis_mcg_auto_trim_exec(unsigned fast_irc)
+{
+    uint32_t expected_count = (CLOCK_BUSCLOCK * 21u);
```

Comment:
Does it make sense to change magic numbers to defines?  I am not too picky if you don't want to though.

Reply:
I don't know if a macro is necessary, maybe adding a comment would explain things enough? The formula is copied straight from the reference manual, which is given without any further explanation what the 21 comes from:

![image](https://user-images.githubusercontent.com/317506/49223582-75897780-f3df-11e8-91ac-9455e1bb68df.png)